### PR TITLE
Improve base styling of PersonaBar & EditBar and allow customization

### LINKED ISSUE #4690: Improve base styling of `PersonaBar` & `EditBar` and allow customization
## Description of enhancement
The base styling of the `PersonaBar` and `EditBar` needs improvement.  Furthermore, it would be wonderful if it was also easily customizable.

## Description of solution
Update base styling of `PersonaBar` and `EditBar`.  Also, @valadas had the wonderful idea of using CSS variables to allow customization. 

## Description of alternatives considered
n/a

## Screenshots
n/a

## Additional context
n/a

## Affected browser
* [x] Chrome
* [x] Firefox
* [x] Safari
* [x] Internet Explorer
* [x] Edge

### PR DESCRIPTION
**Pull request by david-poindexter**
## Summary
Resolves #4690

### CODE REVIEW

File: Dnn.AdminExperience/EditBar/Dnn.EditBar.UI/editBar/css/theme.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/*--------------------------------------------------------------------------------
+These default CSS variables can be overridden and customized by creating a file, 
+containing a copy of the below, called "EditBarTheme.css" and placing it in the 
+"Portals/_default" directory. If the custom CSS file exists, it will be loaded 
+instead of this one.
+---------------------------------------------------------------------------------*/
+:root{
+    --dnn-color-editbar-background: #0b1c24;
```
Comment (mitchelsellers) [inline Dnn.AdminExperience/EditBar/Dnn.EditBar.UI/editBar/css/theme.css:8]: If we are adding an ability to customize with variables, should we expand and allow other changes this way as well?  Foreground color etc

**Review by valadas — APPROVED** [review]
Awesome! Looks all good to me!

File: Dnn.AdminExperience/Library/Dnn.PersonaBar.UI/admin/personaBar/css/theme.css [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+the "Portals/_default" directory. If the custom CSS file exists, it will be loaded 
+instead of this one.
+---------------------------------------------------------------------------------*/
+:root{
+    --dnn-color-pb-menu-background: #0e2936;
+    --dnn-color-pb-menu-background-hover: #0b1c24;
+    --dnn-color-pb-menu-icon: #3c7a9a;
+    --dnn-color-pb-menu-icon-background: #0b1c24;
+    --dnn-color-pb-menu-divider: #1e485e;
+    --dnn-color-pb-menu-text-highlight: #3c7a9a;
```
Comment (mitchelsellers) [inline Dnn.AdminExperience/Library/Dnn.PersonaBar.UI/admin/personaBar/css/theme.css:13]: Same question here for colors etc. being able to change background without foreground colors is limiting.

**Review by mitchelsellers — CHANGES_REQUESTED** [review]

File: Dnn.AdminExperience/EditBar/Dnn.EditBar.UI/editBar/css/theme.css (hunk 1 above)
Comment (david-poindexter) [reply Dnn.AdminExperience/EditBar/Dnn.EditBar.UI/editBar/css/theme.css:8]: Yes, this has great potential for future expansion and greater customization.  This simply establishes the initial base styling goals.

File: Dnn.AdminExperience/Library/Dnn.PersonaBar.UI/admin/personaBar/css/theme.css (hunk 2 above)
Comment (david-poindexter) [reply Dnn.AdminExperience/Library/Dnn.PersonaBar.UI/admin/personaBar/css/theme.css:13]: This achieves all the styling changes as it relates to the initial goals for this solution.  This also handles "foreground colors".  They just aren't called "foreground".

Comment (valadas) [reply Dnn.AdminExperience/Library/Dnn.PersonaBar.UI/admin/personaBar/css/theme.css:13]: IMO, this is just for the actual menu icons and background, if we get into like text foreground colors, then we open a big can of worms into the actual module panels which can have also any number of 3rd party modules. Not saying it's not a good idea, but it's a much much bigger ticket to do right. I would love to do that but in a separate way for the panels (different customization file) later on.

**Review by mitchelsellers — APPROVED** [review]

**Pull request merged**